index.html

```diff
@@ -1,1046 +1,1078 @@
+      mechanism for digitally signing JSON data. It provides a means
+      to ensure the integrity, authenticity, and non-repudiation of
+      the information contained in a JSON object. JWS enables secure
+      communication and trust between parties by adding a
+      cryptographic signature to the JSON payload. This signature is
+      generated using a private key and can be verified using the
+      corresponding public key. JWS ensures the tamper-proof nature of
+      the data, protecting it from unauthorized modifications. The
+      versatility and wide adoption of JWS make it an essential
+      component for verifying the authenticity and integrity of
+      JSON-based messages in various applications, including
+      authentication, authorization, and secure data exchange. These
+      properties make JWS especially well suited to securing the
+      [[VC-DATA-MODEL]] and through this to provide a means of
+      supporting the three party model (issuer, verifier, holder) for
+      exchange of credentials enabled in verifiable credenials.
+    </p>
+    <p>
+      JSON Web Token (JWT) [[RFC7519]] is a widely-used means of
+      expressing
+      <a data-cite="VC-DATA-MODEL#claims">claims</a> to be transferred
+      between two parties. Providing a representation of the
+      Verifiable Credentials Data Model for JWT allows existing
+      systems and libraries to participate in the ecosystem described
+      in Section <a
+      data-cite="VC-DATA-MODEL#ecosystem-overview">ecosystem
+      overview</a>. A JWT encodes a set of
+      <a data-cite="VC-DATA-MODEL#claims">claims</a> as a JSON object
+      that is contained in a JSON Web Signature (JWS) [[RFC7515]]
+      and/or JSON Web Encryption (JWE) [[?RFC7516]]. For this
+      specification, the use of JWE is out of scope.
+    </p>
+    <p>
+      CBOR Object Signing and Encryption (COSE) [[RFC8152]] is a
+      specification that defines a framework for signing and
+      encrypting CBOR (Concise Binary Object Representation) [RFC8949]
+      data structures. COSE provides a standardized way to secure the
+      integrity, authenticity, and confidentiality of CBOR-encoded
+      information. It offers a flexible and extensible set of
+      cryptographic options, allowing for a wide range of algorithms
+      to be used for signing and encryption.
 
+      COSE supports two main operations: signing and encryption. For
+      signing, COSE allows the creation of digital signatures over
+      CBOR data using various algorithms such as HMAC, ECDSA, or
+      EdDSA. These signatures provide assurance of data integrity and
+      authenticity. COSE also supports encryption, enabling the
+      confidentiality of CBOR data by encrypting it with symmetric or
+      asymmetric encryption algorithms. Our use case here, with
+      verifiable credentials is focused on signing, primarily through
+      use of COSE Sign1 operations.
```

Comment:
```suggestion
      asymmetric encryption algorithms.
```

Reply:
Its true that COSE Sign1 is the best way to use COSE to secure vc+ld+json content type, but this seems like not the best place to say this.